openstack_controller/tests/test_openstack_sdk_api.py

```diff
@@ -0,0 +1,491 @@
+import mock
+import pytest
+
+from openstack.exceptions import SDKException
+
+from datadog_checks.openstack_controller.api import OpenstackSdkApi
+from datadog_checks.openstack_controller.exceptions import (AuthenticationNeeded, KeystoneUnreachable,
+                                                            MissingNovaEndpoint, MissingNeutronEndpoint)
+
+
+EXAMPLE_PROJECTS_VALUE = [
```

Comment:
For all these Api output, do you have a link to the doc that can help check out the structure?

Reply:
The [doc](https://docs.openstack.org/openstacksdk/latest/user/connection.html) does not specify the output, I called each methods from the test env to have their explicit output.